.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 80,
+  "semi": false,
```

Comment:
I'd leave that to true, it might become super confusing when coding with standard.js and then use different rules for code inside markdown

Reply:
All the code we have in TypeScript doesn't use semicolons (all the kits in the SDK included)